src/bin/jcli_app/block/genesis/yaml.rs

```diff
@@ -46,27 +42,19 @@ struct BlockchainConfiguration {
     discrimination: Discrimination,
     #[serde(with = "serde::as_string")]
     block0_consensus: ConsensusVersion,
-    slots_per_epoch: Option<u64>,
+    slots_per_epoch: Option<u32>,
     slot_duration: u8,
     epoch_stability_depth: Option<u32>,
     #[serde(default)]
     consensus_leader_ids: Vec<SerdeLeaderId>,
-    consensus_genesis_praos_param_d: Option<SerdeAsString<Milli>>,
     consensus_genesis_praos_param_f: Option<SerdeAsString<Milli>>,
-}
-
-/// the initial configuration of the blockchain
-///
-/// This is the data tha may be updated but which needs
-/// to have an initial value in the blockchain (or not)
-#[derive(Clone, Serialize, Deserialize)]
-struct Update {
     max_number_of_transactions_per_block: Option<u32>,
-    bootstrap_key_slots_percentage: Option<u8>,
+    bft_slots_ratio: Option<SerdeAsString<Milli>>,
```

Comment:
Is this really an optional parameter ? It seems to me that if we start the blockchain without this parameter, the node will fail to start.

Reply:
The default in `Settings::new()` is `Milli::ONE`, so if it's not set in the genesis block, it will get that value. Also, `bft_slots_ratio` is not actually used anywhere at the moment AFAIK.